__tests__/app.test.js

```diff
@@ -229,4 +228,24 @@ describe("NC-Games app", () => {
         });
     });
   });
+  describe("GET /api/reviews", () => {
+    test("Responds with status code 200 and an array.", () => {
+      return request(app)
+        .get(`/api/reviews`)
+        .expect(200)
+        .then(({ body: { reviews } }) => {
+          expect(Array.isArray(reviews)).toBe(true);
+        });
+    });
+    test("Responds with status code 200 and an array of objects sorted by date in descending order.", () => {
+      return request(app)
+        .get(`/api/reviews`)
+        .expect(200)
+        .then(({ body: { reviews } }) => {
+          expect(reviews).toBeSortedBy("created_at", {
+            descending: true,
+          });
```

Comment:
should also be checking if each review has all the keys we expect especially because we need an 'author' key and a 'comment_count'

Reply:
Hey, by 'author' key, do you mean the 'owner' key? I can't see any 'author' key, only 'owner' on the ticket.

Reply:
You're very correct, I should've said 'owner' - thanks for doing those tests!